routers/web/web.go

```diff
@@ -380,15 +388,15 @@ func RegisterRoutes(m *web.Route) {
 	m.Group("/user/settings", func() {
 		m.Get("", user_setting.Profile)
 		m.Post("", bindIgnErr(forms.UpdateProfileForm{}), user_setting.ProfilePost)
-		m.Get("/change_password", auth.MustChangePassword)
-		m.Post("/change_password", bindIgnErr(forms.MustChangePasswordForm{}), auth.MustChangePasswordPost)
+		m.Get("/change_password", userSettingModuleEnabled("password"), auth.MustChangePassword)
+		m.Post("/change_password", userSettingModuleEnabled("password"), bindIgnErr(forms.MustChangePasswordForm{}), auth.MustChangePasswordPost)
 		m.Post("/avatar", bindIgnErr(forms.AvatarForm{}), user_setting.AvatarPost)
 		m.Post("/avatar/delete", user_setting.DeleteAvatar)
 		m.Group("/account", func() {
 			m.Combo("").Get(user_setting.Account).Post(bindIgnErr(forms.ChangePasswordForm{}), user_setting.AccountPost)
 			m.Post("/email", bindIgnErr(forms.AddEmailForm{}), user_setting.EmailPost)
 			m.Post("/email/delete", user_setting.DeleteEmail)
-			m.Post("/delete", user_setting.DeleteAccount)
+			m.Post("/delete", userSettingModuleEnabled("suicide"), user_setting.DeleteAccount)
```

Comment:
Oof, this should not be in this (or any) codebase.
- it's unethical (and sometimes unlawful) to disallow users to delete their own data
- can we maybe not use terminology that confuses killing oneself with deleting ones data?

Reply:
> 

For a public site, I think you are right. But in most cases, the instance maybe set up in a company. It's a work place for employees of that company, so that could disallow users to kill themselves. Since this is an option and default is to allow, I think it's reasonable.

Reply:
I personally **strongly** prefer that Gitea won't include harmful language that are considered trigger words into it's source code. No developer wants a trip down memory lane while browsing the source code.

Reply:
Sorry if the word is not suitable here. Which word do you think it's better?

Reply:
Perhaps `deletion` or `removal`?

Reply:
How about `Delete Your Account` which is the same as words in UI.

Reply:
Done. I chose `deletion`.